stock_picking_report_custom_description/models/sale_order_line.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2023 Tecnativa - Carolina Fernandez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    def write(self, vals):
+        res = super().write(vals)
+        if vals.get("name"):
+            self.env["stock.move"].search(
+                [("sale_line_id", "in", self.ids)]
```

Comment:
I'm now thinking that this should only be applied for pending pickings, not already done ones.

Reply:
@carlosdauden what's your take on this? Do you think it's OK to change done pickings?

Reply:
If the data is changed to improve it, we will certainly want it to be modified everywhere.
If the data is changed to make it worse, we will not want it to be modified in the delivery notes that have already been made (although we would not really want it to change anywhere).

But well, as any modification must be assumed to be made to improve, I prefer that all the pickings are updated.
So that in case the user makes a mistake, he can always rectify it.

Reply:
OK then. Let's keep it.